crates/autopilot/src/run_loop.rs

```diff
@@ -118,7 +119,7 @@ impl RunLoop {
     }
 
     async fn single_run(&self, auction_id: AuctionId, auction: Auction) {
-        tracing::info!(?auction, "solving");
+        tracing::info!(?auction_id, "solving");
```

Comment:
How about specifying all the order uids inside the auction?

Reply:
I think it makes sense to log when new orders were added/removed from the auction (so we can easily find a range of auctions to investigate). Will make a separate PR for that.